doc/examples/transform/plot_transform_types.py

```diff
@@ -0,0 +1,139 @@
+"""
+===================================
+Types of homographies
+===================================
+
+`Homographies <https://en.wikipedia.org/wiki/Homography>`_
+are transformations of a Euclidean space that preserve the alignment of points. 
+Specific cases of homographies correspond to the conservation of more properties,
+such as parallelism (affine transformation), shape (similar transformation) or distances (Euclidean transformation). 
+
+Homographies on a 2D Euclidean space (i.e. for 2D grayscale or multichannel images) are defined by a 3x3 matrix. 
+All types of homographies can be defined either by passing the transformation matrix, or by passing the parameters
```

Comment:
```suggestion
All types of homographies can be defined by passing either the transformation matrix, or the parameters
```
could even drop "passing"